src/components/scenes/LandingScene.tsx

```diff
@@ -11,8 +11,8 @@ import { logEvent } from '../../util/analytics'
 import { scale } from '../../util/scaling'
 import { LogoImageHeader } from '../abSpecific/LogoImageHeader'
 import { BackgroundImage } from '../common/BackgroundImage'
-import { HeaderParentButtons } from '../common/HeaderParentButtons'
```

Comment:
After this change, the `HeaderParentButtons` component will only exist on the PIN login scene. We should finally delete it, maybe in a separate Asana task.